src/components/LoginPage.jsx

```diff
@@ -1,10 +1,15 @@
-// import Box from '@mui/material/Box';
+import CssBaseline from '@mui/material/CssBaseline';
+import Box from '@mui/material/Box';
 import { Button } from '@mui/material';
 import TextField from '@mui/material/TextField';
-// import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
 import { useState } from 'react';
-// import React from 'react';
-import { Link, Route, Routes, useNavigate } from 'react-router-dom';
+import { Route, Routes, useNavigate } from 'react-router-dom';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import '../utils/StyleLoginPage.css';

```

Comment:
It would make sense to colocate the react component file and the styling file that goes with it.